Issue close goes into panic mode when issue number is invalid

### Describe the bug
`gh issue close some_invalid_number` goes into panic mode since the error isn't properly displayed.

gh version :- v0.10.1

### Steps to reproduce the behavior

1. Type this `gh issue close 102154`
2. See error

### Expected vs actual behavior

Expected behaviour
```bash
$ gh issue close 10245
GraphQL error: Could not resolve to an Issue with the number of 10245.
```

Actual behaviour
```bash
$ gh issue close 10245
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0xb4a633]

goroutine 1 [running]:
github.com/cli/cli/command.issueClose(0x13e0640, 0xc00028fae0, 0x1, 0x1, 0x0, 0x0)
        /home/runner/work/cli/cli/command/issue.go:683 +0x573
github.com/spf13/cobra.(*Command).execute(0x13e0640, 0xc00028fac0, 0x1, 0x1, 0x13e0640, 0xc00028fac0)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:842 +0x453
github.com/spf13/cobra.(*Command).ExecuteC(0x13de6c0, 0xc000020090, 0x3, 0x3)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
main.main()
        /home/runner/work/cli/cli/cmd/gh/main.go:55 +0x29c
```
### Logs
Same as mentioned above.

### Extra
I figured out something. There is a code stub that is the root cause I believe:

```go
func issueClose(cmd *cobra.Command, args []string) error {
---Initial stuff---
	issue, err := issueFromArg(apiClient, baseRepo, args[0])  // If there is an error, issue is nil
	var idErr *api.IssuesDisabledError
	if errors.As(err, &idErr) {
		return fmt.Errorf("issues disabled for %s", ghrepo.FullName(baseRepo))
	} else if err != nil {
		return fmt.Errorf("failed to find issue #%d: %w", issue.Number, err)   // We are using issue.Number which causes the nilPointer exception
	}
---Other stuff---
}
```

--- a/command/issue.go
+++ b/command/issue.go
@@ -709,7 +709,7 @@ func issueClose(cmd *cobra.Command, args []string) error {
 	if errors.As(err, &idErr) {
 		return fmt.Errorf("issues disabled for %s", ghrepo.FullName(baseRepo))
 	} else if err != nil {
-		return fmt.Errorf("failed to find issue #%d: %w", issue.Number, err)
+		return err
 	}
 
 	if issue.Closed {
@@ -744,7 +744,7 @@ func issueReopen(cmd *cobra.Command, args []string) error {
 	if errors.As(err, &idErr) {
 		return fmt.Errorf("issues disabled for %s", ghrepo.FullName(baseRepo))
 	} else if err != nil {
-		return fmt.Errorf("failed to find issue #%d: %w", issue.Number, err)
+		return err
 	}
 
 	if !issue.Closed {

